web/src/components/layouts/components/header/header.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsx */
+import { Box } from '@theme-ui/components';
+import React from 'react';
+import { SystemStyleObject } from 'theme-ui';
+import Navbar from './navbar';
+import NavbarMobile from './navbar-mobile';
+
+const styles: SystemStyleObject = {
+  '.navbarMobile': {
+    display: 'none',
+  },
+  ['@media screen and (max-width: 960px)']: {
+    '.navbarMobile': {
+      display: 'block',
+    },
+    '.navbar': {
+      display: 'none',
+    },
+  },
```

Comment:
```suggestion
  '.navbarMobile': {
    display: ['block', 'block', 'none'],
  },
  '.navbar': {
  	display: ['none', 'none', 'block'],
  },
```

Please use [shorthand syntax](https://theme-ui.com/sx-prop#responsive-values).